tests/api2/test_003_network.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+# Author: Eric Turgeon
+# License: BSD
+
+import pytest
+import sys
+import os
+
+apifolder = os.getcwd()
+sys.path.append(apifolder)
+from auto_config import ha, user, password, ip
+from functions import GET, PUT, SSH_TEST
+from config import *
+
+if ha and "domain" in os.environ:
+    domain = os.environ["domain"]
+    gateway = os.environ["gateway"]
+    hostname = os.environ["hostname"]
+    hostname_b = os.environ["hostname_b"]
+    primary_dns = os.environ["primary_dns"]
+    secondary_dns = os.environ["secondary_dns"]
+else:
+    from auto_config import hostname, domain
+
+Reason = "BRIDGEDNS is missing in ixautomation.conf"
+dns_cfg = pytest.mark.skipif("BRIDGEDNS" not in locals(), reason=Reason)
+
+
+@pytest.mark.skipif(not ha and "domain" not in os.environ, reason="Skiping test for Core")
```

Comment:
You have a misspelling `Skiping` needs to be `Skipping`. Looks like this line was copied and pasted a few times so it's misspelled in a few places.